### PR TITLE
Move from com.github.johnynek to org.typelevel.

### PR DESCRIPTION
**Pull request by non**
This commit changes the package in the build, source files, README, docs, etc.

If we wanted to use a different package name from `org.typelevel.packages` as the package then this PR will need to be amended.

### CODE REVIEW

**Comment by non** [comment]
Fixes #34

File: core/src/main/scala/com/github/johnynek/paiges/Doc.scala [hunk 1]
```diff
@@ -619,6 +619,12 @@ object Doc {
     }
 
   /**
+   * Build a document from a single character.
+   */
+  def char(c: Char): Doc =
+    if (c == '\n') Doc.line else Text(new String(Array(c)))
```
Comment (johnynek) [inline core/src/main/scala/com/github/johnynek/paiges/Doc.scala]: we actually have a table of printable ascii characters to reduce allocation. Do you want to use that?

File: core/src/main/scala/com/github/johnynek/paiges/Doc.scala [hunk 2]
```diff
@@ -619,6 +619,13 @@ object Doc {
     }
 
   /**
+   * Build a document from a single character.
+   */
+  def char(c: Char): Doc =
+    if (32 <= c && c <= 126) charTable(c.toInt)
```
Comment (johnynek) [inline core/src/main/scala/com/github/johnynek/paiges/Doc.scala]: don't you need `c.toInt - 32`?

Comment (johnynek) [inline core/src/main/scala/com/github/johnynek/paiges/Doc.scala]: and if this doesn't fail CI, we should add a test for `char(c) == text(c.toString)` I think, and also `char(c).render(0) == c.toString`

Comment (non) [inline core/src/main/scala/com/github/johnynek/paiges/Doc.scala]: Yes!

Comment (non) [inline core/src/main/scala/com/github/johnynek/paiges/Doc.scala]: I'm disturbed that the tests pass -- time to add one.

**Pull request merged**